notify/impl.go

```diff
@@ -606,6 +623,42 @@ func (n *PagerDuty) Notify(ctx context.Context, as ...*types.Alert) (bool, error
 		details[k] = detail
 	}
 
+	images := make([]pagerDutyImage, len(n.conf.Images))
```

Comment:
As `images` and `links` are only used by `notifyV2`, it would make more sense to compute them there. Otherwise LGTM.